src/cljs/bluegenes/pages/regions/views.cljs

```diff
@@ -137,6 +137,17 @@
      [:div.feature-tree-container
       {:class (if @results "shrinkified")} [feature-types-tree]]]))
 
+(defn matches-regex?
+  [v regex]
```

Comment:
The regex works great! Could we add a docstring here explaining what the regex is checking for, in plain english? You can see an example of a docstring on line 152 below.